packages/flutter/lib/src/animation/animations.dart

```diff
@@ -49,7 +60,9 @@ class AlwaysStoppedAnimation extends Animation<double> {
   AnimationDirection get direction => AnimationDirection.forward;
 }
 
+/// A mixin to help with building animations that proxy the value of another animation.
```

Comment:
Add `Implements most of Animation<T>, by deferring their behavior to a given [parent] Animation<double>. To implement an Animation<T> that proxies to a parent, this class plus implementing "T get value" is all that is necessary.` or something.